Kinder/Views/Home/Index.cshtml

```diff
@@ -1,2 +1,58 @@
-﻿
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/mainStyle.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body class="mainBackground">
+    <div>
+        <p class="welcome">
+            Welcome to Kinder! Give away. Get karma points. Compete with other people!
+        </p>
+    </div>
+    <div class="rect">
+        <img src="~/Rectangle 3.png" />
+    </div>
+    <div class="rect1">
+        <img src="~/Rectangle 1.png" />
+    </div>
+    <div class="par4">
+        Each year, a lot of variuos trash is being produced. And sometimes, items that appear in our trash container, are not actually a real trash!
+    </div>
+    <div class="par2">
+        Some people just want to free some space from their variuos items, which are not being used by themselves
+        , but can be useful for some people.
```

Comment:
Some people just want to free up some space by getting rid of their various unused items, but they don't stop to think that their trash could be someone else's treasure.